File: Array&String/flipping_an_image.py
class Solution:
    def flipAndInvertImage(self, image: list[list[int]]) -> list[list[int]]:
        for i in range(len(image)):
            image[i].reverse()
            for j in range(len(image[i])):
                image[i][j] = abs(image[i][j] - 1)
        return image

File: Array&String/CPP/flipping_an_image.cpp
#include <algorithm>
#include <vector>

using namespace std;

class Solution {
public:
    vector<vector<int>> flipAndInvertImage(vector<vector<int>>& image) {
        for (int i = 0; i < image.size(); i++) {
            vector<int> temp = image[i];
            reverse(temp.begin(), temp.end());
            image.at(i) = inverse(temp);
        }
        return image;
    }

    vector<int> inverse(vector<int> arr) {
        for (int i = 0; i < arr.size(); i++) {
            arr[i] = abs(arr[i]-1);
        }
        return arr;
    }
};